### PR TITLE
[CT-1598] chore: Set redact default version to v4

### PR DESCRIPTION
**Pull request by radmiratbrighterai**

### CODE REVIEW

File: redact/__init__.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
 Python client for "brighter Redact"
 """
 
-__version__ = "6.0.0"
+__version__ = "7.0.0"
 
 from .errors import RedactConnectError, RedactResponseError
-from .v3.data_models import (
+from .v3.data_models import JobLabels
```
Comment (pkern90) [inline redact/__init__.py:8]: Do we really need to import v3 here?

**Review by pkern90 — CHANGES_REQUESTED** [review]
Please also add a section to the readme clarifying that it uses redact v4 by default and explaining how to use v3 if really needed. Also pointing out that we recommend switching to v4.

File: redact/__init__.py (hunk 1 above)
Comment (radmiratbrighterai) [reply redact/__init__.py:8]: https://github.com/brighter-ai/redact-client/pull/96/files/1c6774c142a73de4a7cf256820d21ba9d605c914#diff-56b9ba105de1cccc38fa64cf3f185c0ddfcab608c78056dad70ccbc292d4ed3dR31

Comment (radmiratbrighterai) [reply redact/__init__.py]: removed it

File: README.md [hunk 2]
```diff
@@ -109,11 +110,13 @@ from redact import RedactInstance, ServiceType, OutputType
...
 
-In case you use a different api version than the default, you have to import the corresponding classes from the version module instead from `redact`:
+#### Different API version
+In case you use a different api version than the default, you have to import the corresponding classes from the version module instead from `redact`, e.g.:
 
 ```python
-from redact.v4 import RedactInstance, ServiceType, OutputType
+from redact.v3 import RedactInstance, ServiceType, OutputType, JobLabels
 ```
+> IMPORTANT! It is recommended to switch to API version v4, v3 will be removed in the future.
```
Comment (pkern90) [inline README.md:119]: #### Switching API Versions in the client

When your application requires a specific API version that differs from the default provided by the SDK, you must import the relevant classes from that version's specific module. For instance, to use version 3 of the API, your import statement would look like this:

```python
# Importing classes from API version 3
from redact.v3 import RedactInstance, ServiceType, OutputType, JobLabels
```

> 🚨 **IMPORTANT**: API version v3 is currently deprecated and is scheduled for removal in the future. We encourage you to transition to API version v4 promptly to take advantage of the latest features, enhancements, and critical security updates.

**Pull request merged**